src/calendar/DatePicker.tsx

```diff
@@ -105,10 +107,10 @@ export class DatePicker extends ThemedMixin(WidgetBase)<DatePickerProperties> {
 	}
 
 	private _getYearRange() {
-		const { year, yearRange = 20 } = this.properties;
-		const offset = ((year - BASE_YEAR) % yearRange) - yearRange * this._yearPage;
+		const { year, yearRange = 20, yearBasis = DEFAULT_YEAR_BASIS } = this.properties;
```

Comment:
How about defaulting it to the current year?

Reply:
Defaulting the `yearBasis` (or `initialYear`) to the current year would mean that the first year show on a page started with a year +/- the `yearRange`. I think that works well if the date is set to the current year, otherwise the page numbering would seem weird.

i.e. For this year the pages would be:

..., [1999 ... 2018 ], [2019 ... 2038], [2039 ... 2048], ...

![image](https://user-images.githubusercontent.com/331431/70159950-f1fc0980-1676-11ea-931c-5a6d2eeb2082.png)

The `yearBasis` is treated like an offset in starting the year numbering. I would prefer that it was a multiple of 10. I think most people are used to seeing numbers line up in that way.

Reply:
perhaps needs to be the nearest 10 within 10 years of the current year selected then? that would keep the selected value towards the middle. Also, do we really want to expose this to the user via a property? I'd imagine that this would be better kept internal?